src/Traits/ShopOrderIdTrait.php

```diff
@@ -51,8 +51,5 @@ public function setShopOrderId($shopOrderId)
     public function setShopOrderIdResolver(OptionsResolver $resolver)
     {
         $resolver->setAllowedTypes('shop_orderid', ['string', 'int']);
-        $resolver->setAllowedValues('shop_orderid', function ($value) {
-            return preg_match('/^[a-zA-Z0-9 ]{1,100}$/', $value) === 1;
-        });
```

Comment:
Should this really be a free text?

Reply:
@emicha , CMS already provides these restrictions. And as we discussed we should have this validation on the gateway level.